src/detections/configs.rs

```diff
@@ -1018,8 +1018,8 @@ pub struct OutputOption {
     pub exclude_status: Option<Vec<String>>,
 
     /// Only load rules with specific tags (ex: attack.execution,attack.discovery)
-    #[arg(help_heading = Some("Filtering"), long = "tags", value_name = "TAGS", use_value_delimiter = true, value_delimiter = ',', display_order = 460)]
-    pub tags: Option<Vec<String>>,
+    #[arg(help_heading = Some("Filtering"), long = "include_tags", value_name = "TAGS", conflicts_with = "exclude_tags", use_value_delimiter = true, value_delimiter = ',', display_order = 351)]
```

Comment:
```suggestion
    #[arg(help_heading = Some("Filtering"), long = "include-tags", value_name = "TAGS", conflicts_with = "exclude-tags", use_value_delimiter = true, value_delimiter = ',', display_order = 351)]
```

Reply:
@hitenkoku 
Sorry for my misunderstanding!🙇 Where is the `_`(underscore) below defined?
```
      --exclude-tags <TAGS>       Exclude load rules with specific tags (ex: sysmon)
      --include_tags <TAGS>       Only load rules with specific tags (ex: attack.execution,attack.discovery)
```

Reply:
I tried to modify it as you suggested and got it to work, but I made it set both items specified in conflict_with, but I confirmed that the analysis starts as it is without any control.

<img width="776" alt="image" src="https://github.com/Yamato-Security/hayabusa/assets/2350416/8a6c329c-e979-44a1-b34a-be7ecffd47e3">